.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto-merge
+
+on: pull_request_target
+
+
+jobs:
+  auto-merge:
+    if: ${{ github.actor == 'dependabot' }}
```

Comment:
```suggestion
    if: github.actor == 'dependabot'
```

Reply:
Not sure if this should be an actor or some other event field tho.

Reply:
I *think* some events may be caused by somebody else. Like if someone pushes a commit on top of the PR branch. Or when somebody adds a comment. Maybe we need to filter out unwanted events, dunno.

Reply:
Hmm, if we need to add commits etc. then I'm expecting to merge explicitly anyway. I think as long as a newly opened (or automatically rebased) PR from dependabot is merged when CI passes, then it's all good. I think it's probably time to just test it out (I've left a few dependabot PRs open on this repo).

Reply:
Well, this workflow you created is completely separate and does not depend on other CI jobs being green. So it'll get executed asynchronously and will probably just fail to merge even before those tests get a change to get invoked. Unless this action has some wait logic built-in that is...